Project2_2019-08-05.sql

```diff
@@ -0,0 +1,72 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 192.168.20.20 (MySQL 5.6.44)
+# Database: Project2
+# Generation Time: 2019-08-05 15:29:04 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table Plants
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `Plants`;
+
+CREATE TABLE `Plants` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `common_name` varchar(255) NOT NULL DEFAULT '',
+  `latin_name` varchar(255) NOT NULL DEFAULT '',
+  `identification_image` varchar(255) NOT NULL DEFAULT '',
+  `canopy_level` enum('CANOPY LAYER','LOW-TREE LAYER','SHRUB LAYER','HERBACIOUS LAYER','GROUNDCOVER LAYER','RHIZOSPHERE LAYER','VERTICAL LAYER') NOT NULL DEFAULT 'CANOPY LAYER',
+  `life_span` varchar(255) NOT NULL DEFAULT '',
```

Comment:
This isn't very locked down, anything could be entered here at the moment a numerical data type would probably work better